atomic_physics/tests/test_utils.py

```diff
@@ -1,275 +1,205 @@
 import unittest
 
+import scipy.constants as ct
 import numpy as np
-
-from atomic_physics import polarization
+from atomic_physics.ions import ca40, ca43, ba137
+from atomic_physics.utils import (
+    ac_zeeman_shift_for_transition,
+    ac_zeeman_shift_for_state,
+)
 from atomic_physics.core import RFDrive
-from atomic_physics.ions import ba137, ca43
-from atomic_physics.utils import ac_zeeman_shift_for_state
+from atomic_physics.polarization import (
+    PI_POLARIZATION,
+    SIGMA_MINUS_POLARIZATION,
+    SIGMA_PLUS_POLARIZATION,
+)
+
+MU_B = ct.physical_constants["Bohr magneton"][0]
 
 
 def ac_zeeman(Omega, w_transition, w_rf):
     return 0.5 * (Omega**2) * (w_transition / (w_transition**2 - w_rf**2))
 
 
 class TestACZeeman(unittest.TestCase):
-    def test_ac_zeeman_ground_level_ca43(self):
-        """Check AC Zeeman shift calculations using the clock qubit in the 43Ca+ ground
-        level.
+    def test_acz_ca40(self):
         """
-        ion = ca43.Ca43(magnetic_field=146.0942e-4)
-
-        # clock transition
-        upper_state_index = ion.get_state_for_F(level=ca43.S12, F=3, M_F=+1)
-        lower_state_index = ion.get_state_for_F(level=ca43.S12, F=4, M_F=+0)
-        w_transition = ion.get_transition_frequency_for_states(
-            states=(upper_state_index, lower_state_index)
-        )
+        Test the AC Zeeman shift for the ground level qubit in a 40Ca+ ion.
+        """
+        B_lab = 10e-4  # quantization field
+        B_rf = 1e-6  # RF field of 1 uT
+        level = ca40.ground_level
+        Ca40 = ca40.Ca40.filter_levels(level_filter=(level,))
+        ion = Ca40(magnetic_field=B_lab)
 
-        rf_amplitude = 1e-6
-        w_rf = w_transition + 2 * np.pi * 1e6
+        f = abs(ion.get_transition_frequency_for_states((0, 1), relative=True))
+        mu = ion.get_magnetic_dipoles()[0, 1]
+        rabi_freq = mu * B_rf / ct.hbar
+        f_mode = 2 * np.pi * 3e6  # assuming 3 MHz motional mode frequency
+        f_rf = f + f_mode
 
-        # Manually calculate the shifts for each polarization component
-        #
-        # Start by looking at the shift on the (3, +1) state
-        victim_state_index = upper_state_index
+        # Calculate the absolute value of the expected shift for each state
+        # following B.48 in TPH's thesis.

```

Comment:
Let's not cite my thesis for this. We can just call it the Bloch Siegert shift. When I google that, the first hit is wikipedia which has exactly the same expression 
![image](https://github.com/user-attachments/assets/f823cf2d-4645-4503-a989-9abe690169b5)

This is standard enough that we shouldn't need to cite my thesis.

Reply:
This isn't the exact same right?

Reply:
Either way, I've just called it AC Zeeman shift, a quick google of that also gives the required formula :)

Reply:
what's the difference?

Reply:
oh, sorry! you're right, that's the RWA expression